### ISSUE TITLE
[Mobile Feat] Authentication?

### ISSUE THREAD

**Issue by sdtsui** [andrewngu/sound-redux#47]
Heyo. I think it's awesome that this webapp is complete enough that I can start a song on mobile, continue working, and forget that I'm not even using the native app.  Great work!

I don't see a way to log in from the mobile. Can you confirm this? 
If so, I'm also learning React/Redux, and interested in implementing. (See my other issues for other requests)

**Comment by andrewngu** [comment]
Yeah, there is currently no way to authenticate with mobile. I'll admit that mobile was somewhat of an after thought, but I started noticing a lot of traffic on mobile and threw something up super quickly and half assed. If you're interested in implementing, feel free to take a shot at it.

**Comment by mansour-ahmed** [comment]
@sdtsui Have you started working on this issue already? i would be glad to implement that
